ctapipe/reco/hillas.py

```diff
@@ -73,44 +87,84 @@ def hillas_parameters_1(pix_x, pix_y, image):
     image = np.asanyarray(image, dtype=np.float64)
     assert pix_x.shape == image.shape
     assert pix_y.shape == image.shape
-
+    
     # Compute image moments
-    _s = np.sum(image)
-    m_x = np.sum(image * pix_x) / _s
-    m_y = np.sum(image * pix_y) / _s
-    m_xx = np.sum(image * pix_x * pix_x) / _s  # note: typo in paper
-    m_yy = np.sum(image * pix_y * pix_y) / _s
-    m_xy = np.sum(image * pix_x * pix_y) / _s  # note: typo in paper
-
-    # Compute major axis line representation y = a * x + b
-    S_xx = m_xx - m_x * m_x
-    S_yy = m_yy - m_y * m_y
-    S_xy = m_xy - m_x * m_y
-    d = S_yy - S_xx
-    temp = d * d + 4 * S_xy * S_xy
-    a = (d + np.sqrt(temp)) / (2 * S_xy)
-    b = m_y - a * m_x
+    size = np.sum(image)
+
+    #Sanity check1:
+    if size == 0:
+       print ("Empty pixels!")
+       sys.exit(1)
```

Comment:
please change this to raise a `HillasParameterizationError` as in the other sanity checks